homeassistant/components/light/rfxtrx.py

```diff
@@ -94,17 +121,37 @@ def name(self):
         return self._name
 
     @property
+    def isfire_event(self):
+        """ Returns is the device must fire event"""
+        return self._isfire_event
+
+    @isfire_event.setter
```

Comment:
There is no need for this setter, it is set in the constructor and never changed.